tools/pipelines/templates/build-npm-package.yml

```diff
@@ -278,7 +278,7 @@ extends:
                         exit -1;
                       fi
 
-                      if [ -f "lerna.json" ]; then
+                      if [ -f ".releaseGroup" ]; then
```

Comment:
I think this PR is fine with this, it just leaves me a bit concerned for the future. The single-package release groups like api-markdown-documenter don't have either of these files, we still consider them release groups, but for release purposes like here we treat them as individual packages. The logic could get a bit weird in some places. Just something to keep in mind.

Maybe standardizing on always having a packages/ folder, or renaming this file to ".multiPackageReleaseGroup" or something. Don't love the second option :).

Reply:
> The single-package release groups like api-markdown-documenter don't have either of these files, we still consider them release groups

Not exactly... packages like that are "independent packages", and we don't want to treat them like release groups in the pipelines. If/when we get to reworking the pipelines to unify on the release group concept, these .releaseGroup files can likely go away. If needed, we can write a "flub check releaseGroup STRING" command to determine if a path or string name is a valid release group. But I think all these distinctions we make in the pipelines can go away eventually.

Reply:
My bad, you're right, I was mixing release groups and pnpm workspaces. Anyway, looking forward to when the logic becomes simpler 😄 .